packages/@stylexjs/babel-plugin/__tests__/transform-stylex-create-test.js

```diff
@@ -2252,6 +2252,80 @@ describe('@stylexjs/babel-plugin', () => {
           `);
         });
 
+        test('set mixed values', () => {
+          const { code, metadata } = transform(`
+            import * as stylex from '@stylexjs/stylex';
+            export const styles = stylex.create({
+              root: (width) => ({
+                width,
+                backgroundColor: 'red',
+                height: width + 100,
+              })
+            });
+          `);
+          // Check that dynamic number values get units where appropriate
+          expect(code).toMatchInlineSnapshot(`
+            "import * as stylex from '@stylexjs/stylex';
+            export const styles = {
+              root: width => [{
+                kzqmXN: width != null ? "x1bl4301" : width,
+                kWkggS: "xrkmrrc ",
+                kZKoxP: width + 100 != null ? "x1f5funs" : width + 100,
```

Comment:
If `width` is `undefined` then `width + 100` will be `NaN`, and the class will incorrectly get applied